src/main/java/org/opentripplanner/middleware/controllers/api/ApiController.java

```diff
@@ -184,6 +185,11 @@ protected void buildEndpoint(ApiEndpoint baseEndpoint) {
             // OtpUserController. Therefore, the request should be limited to return just the entity matching the
             // requesting user.
             return getObjectsFiltered("_id", requestingUser.otpUser.id);
+        } else if (requestingUser.apiUser != null) {
```

Comment:
Here and line 108 of `AbstractUserController.java`, we may want to add a method on RequestingUser that determines if it is a third party user. For now, that check can just be `apiUser != null`, but it may evolve into some other check (in the event that MOD UI becomes a (first party) API user as well). Not sure if this check needs to go somewhere else that I missed.

Reply:
Updated all occurences.